### DISCUSSION TITLE
firefox: How to use multiple profiles?

### DISCUSSION THREAD

**Discussion by therealkitman** [netblue30/firejail#5964] [Q&A] [answered]
Hi,
I have firefox installed under `/home/chris/firefox/` with a profile under `/data/chris/firefox_profiles/xyz`. I created a .local file under `/home/chris/.config/firejail/firefox.local` with the following and all works ok.

```
ignore apparmor
ignore noexec ${HOME}
whitelist ${HOME}/firefox
whitelist /data/chris/firefox_profiles/xyz
```
But I have a second profile used for banking etc under `/data/chris/firefox_profiles/abc`. How do I call another firefox based local file with these lines (last line is the separate profile) since `home/chris/.config/firejail/firefox.local` already exists?

```
ignore apparmor
ignore noexec ${HOME}
whitelist ${HOME}/firefox
whitelist /data/chris/firefox_profiles/abc
```
Thanks.

**Comment by rusty-snake** [comment]
```bash
firejail --profile=/path/to/second/firefox.local --profile=firefox /usr/bin/firefox
# or
firejail --include=/path/to/second/firefox.local --profile=firefox /usr/bin/firefox
```

However since the locals only different in one command, you do not have to use a secondary .local

```bash
firejail --whitelist=/data/chris/firefox_profiles/abc /use/bin/firefox
```

**Comment by therealkitman** [comment, marked as answer]
The second method of looks good for using in a desktop file.
`firejail --whitelist=/data/chris/firefox_profiles/abc /use/bin/firefox`

However the first method looks interesting. Didn't know you could have multiple `--profile` switches. So would these work as well?
```
firejail --profile=/all/the/same/path/firefox_1.local --profile=firefox /usr/bin/firefox
firejail --profile=/all/the/same/path/firefox_2.local --profile=firefox /usr/bin/firefox
```

**Reply by rusty-snake** [reply]
Try it.

**Comment by therealkitman** [comment]
Tried it! Loved it! Using it! :+1: 

Thank you!!
